plugins/routed-eni/driver/driver_test.go

```diff
@@ -687,7 +683,7 @@ func TestTearDownPodNetwork(t *testing.T) {
 		IP:   net.ParseIP(testIP),
 		Mask: net.IPv4Mask(255, 255, 255, 255),
 	}
-	tearDownNS(addr, 0, mockNetLink)
+	_ = tearDownNS(addr, 0, mockNetLink)
```

Comment:
We probably shouldn't ignore an error. Same with line 714

Reply:
Good point, fixed the test.